ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivCreatePrivacyGroupTest.java

```diff
@@ -44,13 +45,12 @@
   private static final String[] ADDRESSES = new String[] {FROM, "second participant"};
 
   private final Enclave enclave = mock(Enclave.class);
-  private final Enclave failingEnclave = mock(Enclave.class);
   private final PrivacyParameters privacyParameters = mock(PrivacyParameters.class);
+  private final PrivateTransactionHandler privateTransactionHandler =
```

Comment:
Did you want to add a test to say "you can create a group of which you are not a member"? (is this even true?)

Reply:
Don't think it make sense to add this test. Orion will only allow a group to be created is the privateFrom is in the members of the group.